### DISCUSSION TITLE
Angle brackets trigger HTML parsing (was: Specify log level via config)

### DISCUSSION THREAD

**Discussion by alexangas** [YoRyan/mailrise#71] [Q&A]
I'm trying to troubleshoot an issue where apprise is not happy with the contents of the message passed through from mailrise, so I'd like to see details of what's going on.

Mailrise is running via Portainer in a docker container using the published image. Config is specified in a docker compose file and has documented in /etc/mailrise.conf.

Is there a way to specify verbose logging as per #14 via an environment variable or in mailrise.conf ?

**Comment by YoRyan** [comment]
Currently command-line arguments are the only way to control verbosity, but it's [easy to set them](https://docs.docker.com/compose/compose-file/compose-file-v3/#command) via Docker Compose:

```yaml
command: -vv /etc/mailrise.conf
```

**Reply by alexangas** [reply]
Thank you @YoRyan - this was very helpful!

I found that the issue was that the application sending to mailrise was using [this format](https://superuser.com/q/1625686/472) in both the from address and the contents of the email. The angle brackets made apprise think that this was an HTML tag which obviously was invalid. I had tried setting `body_format` to both `text` and `html` in the mailrise config, neither of which worked.

The issue looks to be similar to #1. I don't want to get into whether it should be mailrise or apprise that implements encoding. However I would respectfully suggest that by specifying a value for `body_format` it is easy for the user to assume that mailrise will be ensuring the appropriate encoding or invalid text stripping / replacement such that apprise will accept the payload. 😄

**Reply by YoRyan** [reply]
That's interesting. Certainly if you set `body_format` to `text`, you should expect the HTML not to be parsed. Would it be possible to get a sample message? You may deanonymize it as much as you want.
